react/src/DatePicker/DatePicker.stories.tsx

```diff
@@ -52,5 +52,16 @@ SizeXs.args = {
   defaultValue: new Date('2021-09-13'),
 }
 
+export const DefaultFocusedDate = Template.bind({})
+DefaultFocusedDate.args = {
+  defaultFocusedDate: new Date('2020-09-13'),
+}
+
+export const DefaultFocusedDateOverridenByDefaultValue = Template.bind({})
+DefaultFocusedDateOverridenByDefaultValue.args = {
+  defaultFocusedDate: new Date('2020-09-13'),
+  defaultValue: new Date('2021-09-13'),
+}
+
```

Comment:
these stories are less useful since it requires the datepicker to be opened first and less so for regression testing. Can probably remove first, until design system adds interaction stories.

further reading: https://storybook.js.org/docs/react/writing-stories/play-function